gists/snippets/react.md

```diff
@@ -2,22 +2,22 @@
 
 Defines snippets for common `React` components:
 
-- React-redux components (views)
-- Function UI components
-- Form components
```

Comment:
We shouldn't be encouraging the use of `lp-form` anymore. This will remove the snippet until the #25 PR is merged.